images/nginx/context/etc/nginx/available.d/application.conf

```
@@ -13,7 +13,7 @@ location ~ \.php$ {
     fastcgi_buffers 1024 4k;
     fastcgi_buffer_size 32k;
     fastcgi_busy_buffers_size 256k;
-    fastcgi_read_timeout 60s;
+    fastcgi_read_timeout 3200s;
```

I just checked - this param is set to 600s in Magento2 installation. 10 minutes actually more than enough for debugging purposes. As Magento 1 based config is obviously more older than for Magento 2 i would suggest to replace this param in all places to the following:
```suggestion
    fastcgi_read_timeout 600s;
```

What do you think about that?